src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="container_big"
+      data-qa="big" method="post">
```

Comment:
check out point 5 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)